bigtable_input.py

```diff
@@ -75,6 +76,11 @@
 _game_row_key = re.compile(r'g_(\d+)_m_(\d+)')
 _game_from_counter = re.compile(r'ct_(\d+)_')
 
+# To parse out project, instance, and table from a Bigtable path like:
+#   'projects/tensor-go/instances/minigo-instance/tables/v14-games'
+_unpack_table_path = re.compile(
```

Comment:
Make this a function that does re.match().groups() and returns the value? Then the usage site looks less cluttered.

Reply:
Obviated by the change above (use of a named tuple for the Bigtable spec).